src/components/Time/TimeRangeComponent.tsx

```diff
@@ -77,7 +77,8 @@ export default class TimeRangeComponent extends React.Component<Props> {
   }
 
   renderDuration(d?: DurationInSeconds) {
-    const options = this.props.allowCustom ? { custom: 'Custom', ...serverConfig.durations } : serverConfig.durations;
+    const durations = humanDurations(serverConfig, undefined, 'Traffic');
```

Comment:
This does not work for logs and is not as good for custom runtimes. Change to:
` const durations = humanDurations(serverConfig, 'Last', undefined);`

Reply:
@cfcosta this change is independent of that UX wording issue brought up by Alissa.

Reply:
@jshaughn I changed it to "X Traffic" because that was what was there before. Should I change it either way?

Reply:
Yes, please change it, because before "X Traffic" we had "Last X" and that's what we want, and is part of the motivation of this issue.